generator/src/test/java/com/linkedin/pegasus/generator/TestPegasusDataTemplateGenerator.java

```diff
@@ -205,4 +210,78 @@ private void testRunGenerator(String pegasusFilename, Map<String, String> expect
             file -> file.getName().replace(".java", ""),
             Function.identity()));
   }
+
+  @DataProvider(name = "withMultiDataTemplateCases")
+  private Object[][] createDataTemplateCases()
+  {
+    return new Object[][]
+        {
+            {"IsoDuration.pdsc", "PremiumService.pdsc"},
```

Comment:
can you test using PDL? not to say pdsc is not good for this kind of testing, But since we are migrating to PDL. Need to make sure PDL is actually working. Also if we use PDL now, we don't need to worry about later they are migrated in this MP

Reply:
Yes, PDL cases are added